### PR TITLE
release(0.5.2): 0.5.1 & 0.5.2 Release notes

### PR DESCRIPTION
**Pull request by bryphe**

### CODE REVIEW

File: docs/website/blog/2020-01-06-release-0.5.2.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+Originally, the 0.6.0 release was planned for end of December - so unfortunately I will push it out, as there is still work left to accomplish these renaming goals:
+- VimL plugin integration
+- Snippet support
+- Vim code folding integration
+- Code lens
+- Code actions
+
+I'm going to switch the deadline to "When It's Done" - what I found in 2020
+was that I'd push aggressive deadlines for myself. I believe these deadline / estimates would be in valid in a vaccuum, ie, if I was 100% focused on coding justthose features... but, of course, that's simply never the reality.
```
Comment (CrossR) [inline docs/website/blog/2020-01-06-release-0.5.2.md:145]: `vaccumm` -> `vacuum`, `ie` -> `i.e.`, `justthose` -> `just those`.

File: docs/website/blog/2020-01-06-release-0.5.2.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+- Daily editor blockers / crashes
+- Top upvoted issues
+- Vim workstream
+- VSCode extension host workstream
+
+So once we get to 0.6.0, subsequent milestones will be determined by the highest upvoted issues. This will help to consolidate and stay focused on the right set of priorities for users. You can help me prioritize by upvoting issues that are relevant to you.
+
+Lastly, I've been A/B testing the website - experimenting with Patreon vs the pre-order-lifetime-license. If any patrons would prefer a lifetime license as opposed to a monthly, just send me a message and I'll make sure you're taken care of and get credited the difference.
+
+In the meantine - I'll be continuing to work towards the 0.6.0 release on the timeline -
```
Comment (CrossR) [inline docs/website/blog/2020-01-06-release-0.5.2.md:160]: `meantine` -> `meantime`

File: docs/website/blog/2020-01-06-release-0.5.2.md (hunk 1 above)
Comment (bryphe) [reply docs/website/blog/2020-01-06-release-0.5.2.md]: Ah, thanks - I realized I deleted my spell checker while testing! Re-installing and taking a pass..

File: docs/website/blog/2020-01-06-release-0.5.2.md (hunk 2 above)
Comment (bryphe) [reply docs/website/blog/2020-01-06-release-0.5.2.md]: Thanks!

**Pull request merged**